shared/api/resource.go

```diff
@@ -0,0 +1,29 @@
+package api
```

Comment:
I think this should be moved to server.go

Reply:
Actually, server_resources.go would be more consistent.

Reply:
Right, was about to point that out if we go for `storage_resources.go`. :)

Reply:
obsolete